broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -68,7 +68,12 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
       await engine.createChannels(relayerAddress, balanceToCommit.toString())
     } catch (e) {
       logger.error('Received error when creating outbound channel', { error: e.stack })
-      throw new Error(`Funding error: ${e.message}`)
+
+      if (e && e.message && e.message.includes('too small') && Big(currentBalance).gt('0')) {
```

Comment:
Did something happen where `e` was null? I think you can remove the null check for `e`